daemon/container_operations.go

```diff
@@ -757,6 +757,10 @@ func (daemon *Daemon) connectToNetwork(container *container.Container, idOrName
 
 	container.NetworkSettings.Ports = getPortMapInfo(sb)
 
+	if err := container.ToDisk(); err != nil {
```

Comment:
You moved the `ToDisk()` here, but this function is called in two other places;

- https://github.com/docker/docker/blob/847de59934bd1f9134b235ce70a5c2e1085a470e/daemon/container_operations.go#L515-L517
- https://github.com/docker/docker/blob/847de59934bd1f9134b235ce70a5c2e1085a470e/daemon/container_operations.go#L533-L535

Those locations did not do a `ToDisk()`

Also; looks like you changed `ToDiskLocking()` to `ToDisk()`